radicle/src/cob/issue.rs

```diff
@@ -59,6 +59,21 @@ pub enum State {
     Open,
 }
 
+impl From<String> for State {
+    fn from(s: String) -> Self {
+        match s.as_str() {
+            "closed_other" => Self::Closed {
+                reason: CloseReason::Other,
+            },
+            "closed_solved" => Self::Closed {
```

Comment:
I think we use camelCase in the URL?

Reply:
I think we made the case at some point to use kebap-case for querystring params. Like we do for pagination `?per-page=100`.

Reply:
6ec2614